crates/codegen/grammar/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "codegen_grammar"
```

Comment:
I remember you suggested not using "grammar" as our language definition includes more than the grammar, and we ended up with "language" for the definition, and "schema" for the validation/types of that definition.

Will this crate eventually replace the "codegen_schema" crate? if not, I wonder if we should combine them instead.

Reply:
Yes, and we need to think about naming.

Reply:
Sounds good. I will create the following crates as part of my next PR:

* `codegen/language/macros` for the proc macros
* `codegen/language/definition` for the model + validation

Reply:
nit: in that case, I suggest moving the syntax crates to a `parser` sub-folder as well, matching the rest of the workspace. I'm happy to do it in a later PR as well:

* `codegen/parser/generator`
* `codegen/parser/runtime`

Reply:
done